### DISCUSSION TITLE
Rubrica Final del Proyecto Grupo 7: Auto

### DISCUSSION THREAD

**Discussion by Gona79** [ISPC-TST-EM2022/Grupo7#57] [Announcements]
Nota 1er Semana Grupo 7: 9(Nueve)

Que tal estimado @chulkx, trabajo muy bien 

a) Diagramo el proyecto
b) Perfilo el proyecto temporalmente y por etapas
c) Explico la eleccion del microcontrolador ESP32
d) Realizo la presentacion de su proyecto en formato audiovisual. 
 Quiero felicitarlo por su compromiso y responsabilicdad. Sepa que esto siempre lleva a cosas buenas, pero seguramente un estudiante como Ud esta mas interesado en mejorar y aprender mas, por eso tenga en cuenta el hecho de que los desarrollos electronicos o del IoT debieran estar pensados para soportar ideas y/o procesos en provecho de las personas o sociedad. Por eso tenga presente siempre, que la tecnologia es mejor y mas poderosa cuando se utiliza para soportar ideas en beneficio de las personas y sus procesos, por ejemplo una sencilla y costosa maquina que limpia los pisos en forma automatica. Dentro de estas barredoras en forma de disco y chatas  subyace la estructura de un auto robot como el que esta implementando. Entonces le invito, no es necesario si Ud no lo desea, a pensar en aprovechar su desarrollo para una aplicacion que soporte una idea o proceso en forma practica. Por ejempo, relevar las caracteristicas del  suelo en un campo de siembra. 
Eso por un lado, por otro tenga presente en completar apropiadamente los formularios que subio el profe Jorge Morales @JorEl057. Felicitaciones, continue asi. 

El resto del grupo.: Estimados tengan presente que una sola persona puede llevar adelante cualquier proyecto, la idea de formar grupo cumple muchas funciones: 

 ** Dividir la Carga de las Tareas. 
 ** Adquirir conocimientos practicos especificos ( Sobre Electronica, programacion, diseño)
 ** Vincular el conocimiento para realizar desarrollos. 
 ** Adquirir las herramientas que se utilizan en el ambito profesional. 
 ** Experiencia
 ** Vincularse con otros compañeros del equipo de trabajo en forma colaborativa. 
 ** Obtener un Titulo 

Fijense que la metodologia ABP, en estudiantes mayores, permite que una persona obtenga un titulo sin necesariamente adquirir todo lo otro. Esta en Uds darse cuenta que es mas provechoso.

**Comment by chulkx** [comment]
Buenas tardes profe @Gona79 , muchas gracias por la devolución.
Solo quería hacer un comentario al respecto. Quizás en esta etapa parezca que estuve trabajando solo, ya que los aportes al repositorio así lo indican. Pero al habernos reunido y al haber hecho las aportaciones de cada uno en una plataforma externa a github no se ve reflejado el trabajo real de cada compañero. Por otro lado, por la forma en que dispusimos las tareas que cada uno realizará, en las próximas etapas podrá ver que todos aportamos. Quizás todo esto debimos plasmarlo de antemano, para que usted pudiera verlo.
Procuro, para próximas entregas, presentar las pruebas de las reuniones o del trabajo grupal que hagamos por fuera de esta plataforma, para que todos recibamos igual crédito.
Tambien es cierto que todavía no completamos las planillas que compartió el profe Jorge, es algo que se nos pasó por alto completamente.

**Comment by Gona79** [comment]
Nota de la Semana 3: 10 (diez).

Chicos el objetivo del desarrollo del proyecto era conocer las herramientas de:
Creacion de proyectos ( Gantt, AGILE, Proyectos)
Ideacion, desarrollo y prueba. 
Pero principalmente el trabajo colaborativo. Sigue sin notarse el trabajo colaborativo. Ojo con esto. 
Lo hicieron muy bien, adapten este trabajo en forma de plantilla** y les sera util para el resto de los proyectos de la cursada. Continuen asi.

** Deben crear los readme acordemente. Revisen los otros grupos. Tambien el cierre del proyecto.
